docs/user-guide/aes-operator.md

```diff
@@ -0,0 +1,127 @@
+  * `*` = all versions.
+  * `3.0-ea` = version 3.0-ea1 and any subsequent EA releases on 3.0. Also selects the final 3.0 once the final GA version is released.
+  * `4.*-ea` = version 4.0-ea1 and any subsequent EA release on 4.0. This also selects:
+      * the final GA 4.0.
+      * any incremental and bug fix versions 4.* and 4
+      * the most recent 4.* EA release (i.e., if 4.0.5 is the last GA version and
+        there is a 4.1-EA3, then this selects 4.1-EA3 over the 4.0.5 GA).
+
+Read more about SemVer [here](https://github.com/Masterminds/semver#basic-comparisons).
+
+`updateWindow` is an optional item that will control when the updates can take
+place. This is used to force system updates to happen late at night if that’s
+what the sysadmins want.
+
+There can be any number of `updateWindow` entries (separated by commas).
+Never turns off automatic updates even if there are other entries in the
+comma-separated list. Never is used by sysadmins to disable all updates during
+blackout periods by doing a `kubectl` apply or using our Edge Policy Console to
+set this.
+
+Each `updateWindow` is in crontab format (see https://crontab.guru/) Some
+examples of `updateWindow` are:
+
+* `0-6 * * * SUN`: every Sunday, from 0am to 6am
+* `5 1 * * *`: every first day of the month, at 5am
+
+The Operator cannot guarantee minute time granularity, so specifying a minute in the crontab expression can lead to some updates happening sooner/later than expected.
+
+### Install via Helm Chart
+
+You can also install the AES Operator from a Helm Chart. The following Helm values are supported:
+
+* `image.name`: Operator image name
+* `image.pullPolicy`: Operator image pull policy
+* `namespace`: namespace in which to install the Operator
+
+**To do so:**
+
+1. Add the Helm repository to your Helm client with `helm repo add datawire https://getambassador.io`
+2. Run the following command: `helm install datawire/ambassador-operator`
+3. Once the new Operator is working, create a new CRD called `AmbassadorInstallation` based on the following YAML:
+
+    ```yaml
+    $ cat <<EOF | kubectl apply -n ambassador -f -
+    apiVersion: getambassador.io/v2
+    kind: AmbassadorInstallation
+    metadata:
+      name: ambassador
+    spec:
+      version: 1.1.0
+    EOF
+    ```
+
+## Configuration
+
+After the AmbassadorInstallation is created for the first time, the Operator will then use the list of releases available for the Ambassador Helm Chart for determining the most recent version that can be installed, using the optional Version Syntax for filtering the releases that are acceptable. It will then install Ambassador, using any extra arguments provided in the `AmbassadorInstallation`, like the `baseImage`, the `logLevel` or any of the helmValues.
+
+For example:
+
+```yaml
+$ cat <<EOF | kubectl apply -n ambassador -f -
+apiVersion: getambassador.io/v2
+kind: AmbassadorInstallation
+metadata:
+  name: ambassador
+spec:
+  version: 1.1.0
```

Comment:
Should the example here reference 1.2.0 instead?

Reply:
done